cypress/support/helper.ts

```diff
@@ -53,72 +21,78 @@ export function continuePressingTABKey(count: number, shift = false) {
   }
 }
 
-export function positionOfElement(type: string) {
-  return {
-    first: "0",
-    second: "1",
-    third: "2",
-    fourth: "3",
-    fifth: "4",
-    sixth: "5",
-    seventh: "6",
-    eighth: "7",
-    ninth: "8",
-    tenth: "9",
-    eleventh: "10",
-    thirteenth: "13",
-    nineteenth: "19",
-  }[type];
-}
+const positions = {
+  first: 0,
+  second: 1,
+  third: 2,
+  fourth: 3,
+  fifth: 4,
+  sixth: 5,
+  seventh: 6,
+  eighth: 7,
+  ninth: 8,
+  tenth: 9,
+  eleventh: 10,
+  thirteenth: 12,
```

Comment:
why this is changed from value 13 to 12? I think we a safe to delete those values (because we don't use them)

Reply:
Ah I can remove these two if you would like, as I also noticed they weren't used anywhere in Carbon. I changed these values as, given this function is used to index an array, this wouldn't return the thirteenth and nineteenth elements as arrays are zero-indexed.

Reply:
Now deleted thirteenth and nineteenth 👍🏼